proxy-client/index.ts

```diff
@@ -36,6 +36,37 @@ export interface Diagnostics {
     refsTree: string[];
   };
   peer: {
+    listenAddresses: string[];
```

Comment:
Since we’re not doing anything concrete with the data the following would be acceptable and require less changes in the futures:
```typescript
  peer: unknown
```

```typescript
  peer: zod.unknown()
```

Reply:
👍 14d23ef5de0c29626ff523974aff9c20dd456181